libs/editor/WordPressEditor/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.aztecVersion = 'v1.5.1'
+    ext.aztecVersion = '946-74d200b819d457d42d405a8858665bd7d471acc9'
```

Comment:
Updates aztec to a version which is using the new version of utils which is not dependant on the support library.